test/functional/rpc_deprecated.py

```diff
@@ -7,23 +7,22 @@
 
 class DeprecatedRpcTest(BitcoinTestFramework):
     def set_test_params(self):
-        self.num_nodes = 2
+        self.num_nodes = 1
         self.setup_clean_chain = True
-        self.extra_args = [[], ['-deprecatedrpc=bumpfee']]
+        self.extra_args = [[]]
 
     def run_test(self):
-        # This test should be used to verify correct behaviour of deprecated
-        # RPC methods with and without the -deprecatedrpc flags. For example:
+        # This test should be used to verify the errors of the currently
+        # deprecated RPC methods (without the -deprecatedrpc flag) until
+        # such RPCs are fully removed.
         #
-        # In set_test_params:
-        # self.extra_args = [[], ["-deprecatedrpc=generate"]]
-        #
-        # In run_test:
         # self.log.info("Test generate RPC")
         # assert_raises_rpc_error(-32, 'The wallet generate rpc method is deprecated', self.nodes[0].rpc.generate, 1)
-        # self.generate(self.nodes[1], 1)
+        #
+        # Please ensure that the -deprecatedrpc flag for this RPC method is
+        # used in another test where the RPC functionality is actually tested.
```

Comment:
nit: Needed to read this comment several times before understanding, suggested small change for clarity. 
```suggestion
        # Please ensure that the -deprecatedrpc flag for all the RPC methods
        # mentioned above is used in their respective functional tests, 
        # where the RPC functionality is actually tested.
```

Reply:
I have reworked that part a bit, I hope it's clearer now.